From 4c40289eb9e47cfd272a8cc402fd2ddb29e2a3dc Mon Sep 17 00:00:00 2001
From: Alexander Aring <aahringo@redhat.com>
Date: Wed, 24 May 2023 13:50:59 +0000
Subject: [PATCH] dlm_controld: remove unnecessary header include

The timewarn netlink functionality got dropped and will be removed by
kernel v6.4. The user space part was already dropped by commit 34ea31e7
("controld: remove timewarn handling"). This is just a left over of this
commit. Recent builds fails now because the UAPI header in the Linux
kernel was removed. This means older dlm sources cannot be build with
newer kernel-headers, however it is not recommended to use older dlm
sources and all existing users should upgrade anyway.

Upstream-Status: Backport [https://pagure.io/dlm/c/ddbba6608896f81bfce8f8edf3d0f507714cfc43?branch=main]
Reported-by: Fabio M. Di Nitto <fdinitto@redhat.com>
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 dlm_controld/main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dlm_controld/main.c b/dlm_controld/main.c
index 7cf6348..e70e96a 100644
--- a/dlm_controld/main.c
+++ b/dlm_controld/main.c
@@ -12,7 +12,6 @@
 #include <pthread.h>
 #include <linux/netlink.h>
 #include <linux/genetlink.h>
-#include <linux/dlm_netlink.h>
 #include <uuid/uuid.h>
 
 #ifdef USE_SD_NOTIFY
-- 
2.41.0

